.changie.yaml

```diff
@@ -78,15 +79,22 @@ custom:
 
 footerFormat: |
   {{- $contributorDict := dict }}
-  {{- /* any names added to this list should be all lowercase for later matching purposes */}}
-  {{- $core_team := list "michelleark" "peterallenwebb" "emmyoop" "nathaniel-may" "gshank" "leahwicz" "chenyulinx" "stu-k" "iknox-fa" "versusfacit" "mcknight-42" "jtcohen6" "aranke" "dependabot[bot]" "snyk-bot" "colin-rogers-dbt" }}
+  {{- /* ensure all names in this list are all lowercase for later matching purposes */}}
+  {{- $exclude_authors := list "dependabot[bot]" "snyk-bot"}}
+  {{- $core_team := splitList " " .Env.CORE_TEAM }}
+  {{- $core_team_lower := list }}
+  {{- range $team_member := $core_team }}
+    {{- $team_member_lower := lower $team_member }}
+    {{- $core_team_lower = append $core_team_lower $team_member_lower }}
+  {{- end }}
+  {{- $exclude_authors := concat $exclude_authors $core_team_lower}}
```

Comment:
2 minor nitpicks:

1) Could we append $team_member_lower directly to $exclude_authors? Then we wouldn't need to initialize core_team_lower on 85 nor concat them on 90.
2) `authors` suggests the PR author to me; but we're excluding are the maintainers. Could we name that list $exclude_maintainers, or simply $maintainers? line 97 makes more sense to me in isolation with the latter name.